tests/phpunit/tests/xmlrpc/wp/newComment.php

```diff
@@ -48,12 +74,18 @@ function test_new_comment_duplicated() {
 		$this->make_user_by_role( 'administrator' );
 		$post = self::factory()->post->create_and_get();
 
-		$comment_args = array( 1, 'administrator', 'administrator', $post->ID, array(
-			'content' => rand_str( 100 ),
-		) );
+		$comment_args = array(
+			1,
+			'administrator',
+			'administrator',
+			$post->ID,
+			array(
```

Comment:
Should be on one line since it's only one item.

Reply:
and the one above also since it's not an associative array